include/boost/geometry/algorithms/detail/expand/box.hpp

```diff
@@ -51,9 +51,11 @@ struct expand
     template <typename Strategy>
     static inline void apply(BoxOut& box_out,
                              BoxIn const& box_in,
-                             Strategy const& )
+                             Strategy const& strategy)
     {
-        Strategy::apply(box_out, box_in);
+		// strategy.expand(box_out, box_in).apply(box_out, box_in);
+		using strategy_t = decltype(strategy.expand(box_out, box_in));
+		strategy_t::apply(box_out, box_in);
```

Comment:
In the past I opposed to the `_t` suffix, but I'm OK with it now. It's standard. No objection.